src/main/java/controller/LottoController.java

```diff
@@ -21,16 +24,18 @@ public void play() {
 		displayLottoGameResult(lottoGameResult);
 	}
 
-	private static void displayLottoGameResult(LottoGameResult lottoGameResult) {
-		LottoView.displayResult(lottoGameResult);
-		LottoView.displayYield(lottoGameResult.calculateYield());
-	}
+	private static LottoTickets buyLottoTickets() {
+		Cash cash = LottoView.getCash();
+		int countOfManualTickets = LottoView.getCountOfManualTickets();
+		cash.purchase(LottoTicket.PRICE, countOfManualTickets);
+		LottoTickets lottoTickets = LottoView.getManualLottoTickets(countOfManualTickets);
 
-	private static LottoGameResult runLottoGame(LottoTickets lottoTickets,
-		WinningLottoTicket lottoWinningNumbers) {
-		LottoGame lottoGame = new LottoGame(lottoTickets, lottoWinningNumbers);
-		LottoGameResult lottoGameResult = lottoGame.calculateResult();
-		return lottoGameResult;
+		int countOfRandomTickets = cash.getTicketCount();
+		lottoTickets.addRandomLottoTicket(countOfRandomTickets);
```

Comment:
기존의 lottoTickets에 새로운 lottoTickets들을 add하는 것 보다는 
두 lottoTickets를 합쳐서 새로운 LottoTickets으로 만드는 방식으로 변경해보면 어떨까요? 😃


```suggestion
		LottoTickets totalLottoTicket = lottoTickets.plus(countOfRandomTickets);
		
		...
		return totalLottoTicket;
```

Reply:
LottoTickets를 불변객체로 만들라는 말씀이시군요?
반영했습니다. https://github.com/next-step/java-lotto-kakao/pull/35/commits/113c2bf2fe873db2007dcf275c5f59d92951a4d5